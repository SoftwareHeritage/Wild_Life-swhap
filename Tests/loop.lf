count :-
        N=0,
	repeat,
        write('What was ',N),
        N <<- N+1,
        write(' becomes ',N),
        N=@(abc,def,N-7,N*5),
        write(' then ',N),
        nl,
        N>30.


