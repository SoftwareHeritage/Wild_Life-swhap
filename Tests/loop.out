> 
*** Yes
> > What was 0 becomes 1 then 1(abc,def,-6,5)
What was 1 becomes 2 then 2(abc,def,-5,10)
What was 2 becomes 3 then 3(abc,def,-4,15)
What was 3 becomes 4 then 4(abc,def,-3,20)
What was 4 becomes 5 then 5(abc,def,-2,25)
What was 5 becomes 6 then 6(abc,def,-1,30)
What was 6 becomes 7 then 7(abc,def,0,35)
What was 7 becomes 8 then 8(abc,def,1,40)
What was 8 becomes 9 then 9(abc,def,2,45)
What was 9 becomes 10 then 10(abc,def,3,50)
What was 10 becomes 11 then 11(abc,def,4,55)
What was 11 becomes 12 then 12(abc,def,5,60)
What was 12 becomes 13 then 13(abc,def,6,65)
What was 13 becomes 14 then 14(abc,def,7,70)
What was 14 becomes 15 then 15(abc,def,8,75)
What was 15 becomes 16 then 16(abc,def,9,80)
What was 16 becomes 17 then 17(abc,def,10,85)
What was 17 becomes 18 then 18(abc,def,11,90)
What was 18 becomes 19 then 19(abc,def,12,95)
What was 19 becomes 20 then 20(abc,def,13,100)
What was 20 becomes 21 then 21(abc,def,14,105)
What was 21 becomes 22 then 22(abc,def,15,110)
What was 22 becomes 23 then 23(abc,def,16,115)
What was 23 becomes 24 then 24(abc,def,17,120)
What was 24 becomes 25 then 25(abc,def,18,125)
What was 25 becomes 26 then 26(abc,def,19,130)
What was 26 becomes 27 then 27(abc,def,20,135)
What was 27 becomes 28 then 28(abc,def,21,140)
What was 28 becomes 29 then 29(abc,def,22,145)
What was 29 becomes 30 then 30(abc,def,23,150)
What was 30 becomes 31 then 31(abc,def,24,155)

*** Yes
--1> 
